docs/ImportantPythonFunctions.md

```diff
@@ -0,0 +1,180 @@
+# Important Python Functions in ONNX
+
+## Loading an ONNX Model
+```python
+import onnx
+
+onnx_model = onnx.load('path/to/the/model')
+```
+Runnable IPython notebooks:
+- [load_model.ipynb](https://github.com/onnx/onnx/tree/master/onnx/examples/load_model.ipynb)
+
+## Saving an ONNX Model
+```python
+import onnx
+
+onnx_model = ... # Your model in memory
+
+# Save the ONNX model
+with open('path/to/the/model', 'wb') as f:
+    f.write(onnx_model.SerializeToString())
+```
+Runnable IPython notebooks:
+- [save_model.ipynb](https://github.com/onnx/onnx/tree/master/onnx/examples/save_model.ipynb)
+
+## Manipulating TensorProto and Numpy Array
+```python
+import numpy
+import onnx
+import os
+from onnx import numpy_helper
+
+# Preprocessing: create a Numpy array
+numpy_array = numpy.array([[1.0, 2.0, 3.0], [4.0, 5.0, 6.0]], dtype=float)
+print('Original Numpy array:\n{}\n'.format(numpy_array))
+
+# Convert the Numpy array to a TensorProto
+tensor = numpy_helper.from_array(numpy_array)
```

Comment:
`from_array` takes an optional argument `name`.

Reply:
Sure, will cover this case. :-)